python/ray/autoscaler/aws/example-gpu-docker.yaml

```diff
@@ -103,25 +103,22 @@ file_mounts: {
 #    "/path2/on/remote/machine": "/path2/on/local/machine",
 }
 
-# List of shell commands to run to set up nodes.
-setup_commands:
+# List of commands that will be run each time the node is booted.
+# On the first boot, these will run after `node_creation_commands`.
+node_start_commands:
     # - pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev6-cp27-cp27mu-manylinux1_x86_64.whl
-    - pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev6-cp35-cp35m-manylinux1_x86_64.whl
+    - IN_DOCKER pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev6-cp35-cp35m-manylinux1_x86_64.whl
```

Comment:
To make it more clear IN_DOCKER is a magic command, how about $IN_DOCKER instead (or something else if it doesn't really have env var semantics).

Reply:
It doesn't quite have environment variable semantics since the following string gets *quoted*, and then the `docker exec ...` command is prepended. In this sense it feels more like a command or a bash function to me, so I'm uneasy about `$IN_DOCKER` (also would we then need to handle alternative expansions, like `${IN_DOCKER}`?)

Certainly open to alternatives though. Perhaps adding a special symbol would help, e.g. `@IN_DOCKER`? Or even just a more explicit name like `RUN_IN_DOCKER` or `DOCKER_EXEC`.

Reply:
@IN_DOCKER sounds good.